doc/tutorials/starship.rst

```diff
@@ -0,0 +1,99 @@
+.. _starship:
+
+Running Jobs with StarshipEngine
+################################
+
+.. sectionauthor:: Zeid Zabaneh <zeid@xanadu.ai>
+
+In this section, we provide a tutorial of the **StarshipEngine**, an engine used to connect to the Strawberry Fields platform and execute jobs remotely (e.g. on a quantum chip).
+
+Configuring StarshipEngine
+==========================
+
+Before using StarshipEngine, you need to configure the hostname and authentication token that will provide you access to the API. The easiest way is to create a configuration file named ``config.toml`` in your working directory. A typical file looks like this:
+
+.. code-block:: console
+
+   [api]
+   hostname = "platform.example.com"
+   authentication_token = "ElUFm3O6m6q1DXPmpi5g4hWEhYHXFxBc"
+
+You can generate this file interactively by using the ``starship`` command as follows, answering the questions in the prompts.
+
+.. code-block:: console
+
+   (strawberryfields)$ starship --reconfigure
```

Comment:
What is `(strawberryfields)`? an environment?

Reply:
Yes - I will change this to be more generic though, because it doesn't make sense this way.

Reply:
It might be worth removing the `$` as well. We used to use it everywhere to denote a shell environment (and I think it is still used on the installation page), but I've had some users copy and paste the dollar sign thinking it was part of the command.